smdebug/core/utils.py

```diff
@@ -590,3 +591,52 @@ def check_smdataparallel_env():
             _smdataparallel_imported = None
 
     return _is_invoked_via_smddp
+
+
+def check_smmodelparallel_training():
```

Comment:
Now that we're creating this helper function, can we update the SMP check in line 425 in `get_distributed_worker` to use this function? https://github.com/awslabs/sagemaker-debugger/pull/488/files#diff-ab89aaf29e212f79957b4dcbd005ac9e44352f208b3fb250003f75b964a2fd22R425

Reply:
The get_distributed_worker is getting the number of workers from the imported module.  The current function inspects whether the job is model parallel or not.  I would like to keep the scope of this change limited to engage/disengage behavior.